includes/documents/class-wcpdf-invoice.php

```diff
@@ -364,6 +364,22 @@ public function get_pdf_settings_fields( $option_name ) {
 					'description'	=> __( 'Enter the number of days for the due date.', 'woocommerce-pdf-invoices-packing-slips' ),
 				)
 			),
+			array(
+				'type'          => 'setting',
+				'id'            => 'due_date_base_date',
+                'title'         => __( 'Due date base date', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I would avoid repeating the word **date**. **Due date base** seems good to me. What you think?

Reply:
Yeah, it seems good.